library/std/src/sys/pal/unix/process/process_unix.rs

```diff
@@ -19,8 +19,7 @@ use crate::sys::process::process_common::*;
 use crate::{fmt, mem, sys};
 
 cfg_if::cfg_if! {
-    // This workaround is only needed for QNX 7.0 and 7.1. The bug should have been fixed in 8.0
-    if #[cfg(any(target_env = "nto70", target_env = "nto71", target_env = "nto71_iosock"))] {
+    if #[cfg(any(target_env = "nto70", target_env = "nto71", target_env = "nto71_iosock", target_env = "nto80"))] {
```

Comment:
https://www.qnx.com/developers/docs/8.0/com.qnx.doc.neutrino.lib_ref/topic/f/fork.html still mentions the same issue

Reply:
The changes of this file really make me thinking: If the bugfix is required for all recent QNX version, including the most recent one (and it will take 2-3 years for the next release if they keep the same schedule as before), then why don't we change it to `target_os = "nto"`?

Reply:
@flba-eb agreed, we can keep it "nto" till a fix is available.